linode/manager
# fix: [M3-7151] - Redis trademark changes

## Description 📝
Added Redis Legal verbiage to the app descriptions and any instance of Redis includes the Trademark logo

## Preview 📷

![image](https://github.com/linode/manager/assets/115299789/d432fbf0-9cd6-4b32-a613-cbede9608d37)

| Single node  | Cluster   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/115299789/b2488117-0443-4e8e-a46d-0ac74b524b8b) | ![image](https://github.com/linode/manager/assets/115299789/65674295-1cd6-4bb5-af0f-8c8690490c8e) |

## How to test 🧪
- Go to `/linodes/create?type=One-Click`
- Ensure mentions of redis include a trademark
- Ensure searching for redis still displays the two apps
- Ensure the drawer still opens and that there is a legal disclaimer in the drawer
- Redis logo should be replaced by 3 generic stacked slices
